apps/sv/src/main/openapi/sv-internal.yaml

```diff
@@ -233,6 +233,27 @@ paths:
                 "$ref": "#/components/schemas/GetSynchronizerNodeIdentitiesDumpResponse"
         "500":
           $ref: "../../../../common/src/main/openapi/common-external.yaml#/components/responses/500"
+  /v0/admin/domain/export-snapshot:
```

Comment:
I just discovered we do actually have
```
    post:
      tags: [ sv ]
      x-jvm-package: sv_admin
      operationId: "triggerDomainMigrationDump"
      requestBody:
        required: true
        content:
          application/json:
            schema:
              "$ref": "#/components/schemas/TriggerDomainMigrationDumpRequest"
      responses:
        "200":
          description: ok
        "500":
          $ref: "../../../../common/src/main/openapi/common-external.yaml#/components/responses/500"
```
Haven't used it in years but does this already do what we need?